scripts/cloudbuild_config.json

```diff
@@ -0,0 +1,18 @@
+{
+  "gpu_versions" : {
+    "100" : ["CUDA=10.0"],
+    "101" : ["CUDA=10.1"]
+  },
+  "cpu_versions" : {
+    "py37" : ["PYTHON_VERSION=3.7", "MINICONDA_VERSION=py37_4.8.2"],
+    "py38" : ["PYTHON_VERSION=3.8", "MINICONDA_VERSION=py38_4.8.2"]
+  },
+  "images" : [
+    {"cpu" : "py37"},
```

Comment:
why is the python version a value, keyed by `cpu` here?

Reply:
Is the concern here that the 'cpu' key is misleading? Easy enough to rename this to 'python' if that helps.